source/includes/_evidence.md

```diff
@@ -226,7 +226,7 @@ curl https://api.gresb.com/api/v0/entities/{entity_id}/responses/{response_id}/d
 }
 ```
 
-## POST values/{variable} - Add Evidence Notes
+## POST values/{variable} - Add Evidence Notes for File
```

Comment:
It is indeed the same, but titles with the same name were breaking the navigation in the menu so I changed the names to be different: _Add Evidence Notes for Hyperlink_ and _Add Evidence Notes for File_. Another way to handle this would be to leave only one section for Notes, but that doesn't fit in the current structure.